src/Tests/dotnet.Tests/CommandTests/CompleteCommandTests.cs

```diff
@@ -275,5 +276,35 @@ public void GivenNuGetTrustCommandItDisplaysCompletions()
             CompleteCommand.RunWithReporter(new[] { "dotnet nuget trust " }, reporter).Should().Be(0);
             reporter.Lines.OrderBy(c => c).Should().Equal(expected.OrderBy(c => c));
         }
+
+        [Fact]
+        public void GivenNuGetSignCommandItDisplaysCompletions()
+        {
+            var expected = new[] {
+                "--output",
+                "--certificate-path",
+                "--certificate-store-name",
+                "--certificate-store-location",
+                "--certificate-subject-name",
+                "--certificate-fingerprint",
+                "--certificate-password",
+                "--hash-algorithm",
+                "--timestamper",
+                "--timestamp-hash-algorithm",
+                "--verbosity",
```

Comment:
By looking at existing code it looks we need to order by alphabetic order.
`--output` here.

Reply:
Thanks. Fixed the order.
I think the sequence doesn't affect the testing results, as we will sort both the real results and the expected results.